modules/ProjectAPI/src/test/java/org/gephi/project/io/XMLCharTest.java

```diff
@@ -0,0 +1,13 @@
+package org.gephi.project.io;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class XMLCharTest {
+	@Test
+	public void isValidInputPositiveOutputFalse() {
```

Comment:
Either make this into two tests (true and false) or rename the test to something like `testIsValid()`
Currently, the name contains `False` but both true and false outcomes are expected.